MSW/MSW/AvailableSimulation.cs

```diff
@@ -29,7 +29,8 @@ public AvailableSimulation(SimulationConfig a_config)
 		private void DiscoverAvailableVersions()
 		{
 			string version = LatestVersionName;
-			string versionFile = m_config.SimulationName + @"data\version.txt";
+			var separator = Path.DirectorySeparatorChar;
+			string versionFile = m_config.SimulationName + $"data{separator}version.txt";
```

Comment:
it's fine . But for consistency you could argue to just use / like in the consolelogger lines ?

Reply:
(my) Windows didn't seem to like it...